randovania/data/json_data/prime2.txt

```diff
@@ -4270,26 +4480,27 @@ Asset id: 3436835742
           Morph Ball
           Any of the following:
               Spider Ball and After Dark Samus 1 and After Main Reactor Reloaded after DS1
-              Morph Ball Bomb and Space Jump Boots and Bomb Space Jump (Veteran) and Difficulty: Veteran
+              All of the following:
+                  Morph Ball Bomb and Bomb Space Jump (Veteran) and Difficulty: Veteran
+                  Use Screw Attack (Space Jump)
   > Main Reactor/Door to Security Station B
       Any of the following:
-          Dark Beam
-          Morph Ball and Morph Ball Bomb and Space Jump Boots and After Dark Samus 1 and Bomb Space Jump (Hard) and Difficulty: Hard
-          Space Jump Boots and Screw Attack and After Dark Samus 1 and Slope Jump (Veteran) and Difficulty: Veteran
-          Space Jump Boots and Screw Attack and Difficulty: Trivial
-          Screw Attack and Screw Attack without Space Jump (Trivial) and Difficulty: Trivial
-  > Main Reactor/Before Pickup
-      All of the following:
-          After Dark Samus 1 and After Main Reactor Reloaded after DS1
-          Any of the following:
-              Dark Beam and Space Jump Boots and Screw Attack and Roll Jump (Hypermode) and Difficulty: Hypermode
-              Morph Ball and Boost Ball and Spider Ball
-              Morph Ball and Spider Ball and Morph Ball Bomb and Space Jump Boots and Difficulty: Hard
-              Morph Ball and Spider Ball and Morph Ball Bomb and Difficulty: Hypermode
-              Morph Ball and Morph Ball Bomb and Space Jump Boots and Bomb Space Jump (Veteran) and Difficulty: Veteran
-              Morph Ball and Space Jump Boots and Screw Attack and Roll Jump (Trivial) and Difficulty: Hypermode
-              Space Jump Boots and Screw Attack and Slope Jump (Veteran) and Difficulty: Veteran
-              Space Jump Boots and Screw Attack and Difficulty: Hard
+          All of the following:
+              Dark Beam
+              Charge Beam or Dark Ammo
+          All of the following:
+              Knowledge (Trivial) and Difficulty: Trivial
+              Any of the following:
```

Comment:
This one here seems weird.

Reply:
It's meant to be either dark beam or screw attack, but I think the screw attack grouping is incorrect.

Reply:
Looking it over, it doesn't seem weird to me. It's any of the following 3 different scenarios for Storage D to Security Station B:
- Dark Beam for the gate
- Screw Attack for the gate
- BSJ to leave without Dark Beam or Screw Attack

First 2 are for anytime, 3rd is for after DS1

Reply:
It's just that the screw attack alternative can be reduced to this: `Knowledge (Trivial) and Difficulty: Trivial and Use Screw Attack (Space Jump) and (Screw Attack without Space Jump (Trivial) or Use Screw Attack (No Space Jump))`

This implies that Space Jump is always required, so all the "without SJ" alternatives is pointless.